app/routes/connectors/apps/formBuilder.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+
+import { capitalize, lowerCase } from 'lodash';
+import { Controller } from 'react-hook-form';
+
+import { TextField } from '@numaryhq/storybook';
+
+import i18n from '~/src/translations';
+import {
+  ConnectorConfigFormProps,
+  InputType,
+} from '~/src/types/connectorsConfig';
+import { ObjectOf } from '~/src/types/generic';
+
+export const connectorsConfig = {
```

Comment:
add a TODO to explain that this config will be consumed from backend when it will be ready to do so